File: Order.DAL/Entities/Order.cs
using Order.DAL.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Order.DAL.Entities
{
    public class Order : BaseEntity
    {
        public DateTime OrderDate { get; set; }
        public OrderStatus OrderStatus { get; set; }
        public int CustomerId { get; set; }
        public int VehicleId { get; set; }
        public int RouteId { get; set; }
        public Route Route { get; set; }
        public int JourneyId { get; set; }
        public Journey Journey { get; set; }
    }
}

File: Client/angular-app/src/app/core/models/order.ts
import { journey } from "./journey";
import { orderStatus } from "./orderStatus";
import { route } from "./route";
import { vehicleShort } from "./vehicleShort";

export class order {
    id: number = 0;
    placeDate: string = "";
    status: orderStatus = 0;
    userId: string = "";
    vehicle: vehicleShort = new vehicleShort();
    route: route = new route();
    journey: journey = new journey();
}